test/plug/router_test.exs

```diff
@@ -1,3 +1,5 @@
+Application.ensure_all_started(:telemetry)
```

Comment:
This should not be necessary, Mix should automatically start it. :)

Reply:
I added that when these tests blew up because of a badarg when calling telemetry.